controller/array_action/array_mediator_ds8k.py

```diff
@@ -158,7 +158,7 @@ def disconnect(self):
         pass
 
     def is_active(self):
-        return True
+        return self.client.is_valid()
```

Comment:
why `valid` and not `active`?

Reply:
because it can be with illegal password which means it is active but invalid

Reply:
but, if I understand correctly, `is_active` currently returns `False` on invalid password exception

Reply:
yes. Because we can't really check is active for ds8k so instead we assume it is active if and only if it is valid.

Reply:
why can't we check in ds8k if the exception is due to invalid credentials?

Reply:
as i wrote - we can't check if token is not up-to-date
and anyway i want to handle any unexpected error

Reply:
if token is not up-to-date, what does it mean in terms of active and valid?

Reply:
and do "invalid password" and "expired token" raise the same exceptions?

Reply:
1) I'm not entering discussion about method name. 
2) I don't know which exception is raised

Reply:
so could you just add a ticket number for after GA to resolve remaining conversations here?